Gemfile.lock

```diff
@@ -0,0 +1,171 @@
+GIT
+  remote: git@github.com:indiegogo/active_model_serializers.git
+  revision: 91352cf8f900f3be9ff07cd722ac00d826803b27
+  branch: 0-8-stable
+  specs:
+    active_model_serializers (0.8.1)
+      activemodel (>= 3.0)
+
+PATH
+  remote: ..
+  specs:
+
+PATH
+  remote: .
+  specs:
+    rocket_pants (5.0.0)
+      actionpack (>= 4.0, < 6.0)
+      api_smith
+      hashie (>= 1.0, < 4)
+      moneta
+      railties (>= 4.0, < 6.0)
+      will_paginate (~> 3.0)
+
+GEM
+  remote: https://rubygems.org/
+  specs:
+    actionpack (5.0.0)
+      actionview (= 5.0.0)
+      activesupport (= 5.0.0)
+      rack (~> 2.0)
+      rack-test (~> 0.6.3)
+      rails-dom-testing (~> 2.0)
+      rails-html-sanitizer (~> 1.0, >= 1.0.2)
+    actionview (5.0.0)
+      activesupport (= 5.0.0)
+      builder (~> 3.1)
+      erubis (~> 2.7.0)
+      rails-dom-testing (~> 2.0)
+      rails-html-sanitizer (~> 1.0, >= 1.0.2)
+    activemodel (5.0.0)
+      activesupport (= 5.0.0)
+    activerecord (5.0.0)
+      activemodel (= 5.0.0)
+      activesupport (= 5.0.0)
+      arel (~> 7.0)
+    activesupport (5.0.0)
+      concurrent-ruby (~> 1.0, >= 1.0.2)
+      i18n (~> 0.7)
+      minitest (~> 5.1)
+      tzinfo (~> 1.1)
+    addressable (2.5.2)
+      public_suffix (>= 2.0.2, < 4.0)
+    api_smith (1.3.0)
+      hashie (>= 1.0, < 3.0)
+      httparty
+    arel (7.1.4)
+    builder (3.2.3)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    concurrent-ruby (1.0.5)
+    crack (0.4.3)
```

Comment:
I thought you were removing `crack`, according to the comment in the `CHANGELOG.md`.

Reply:
it looks like webmock uses crack unfortunately